### PR TITLE
docs: Add README to Solve Too Many Openfiles in MacOS

### PR DESCRIPTION
**Pull request by irvifa**
This should solve #22895

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,51 @@ make serve
...
+Environment: "development"
+Serving pages from memory
+Running in Fast Render Mode. For full rebuilds on change: hugo server --disableFastRender
+Web Server is available at http://localhost:1313/ (bind address 127.0.0.1)
+Press Ctrl+C to stop
+ERROR 2020/08/01 19:09:18 Error: listen tcp 127.0.0.1:1313: socket: too many open files
+make: *** [serve] Error 1
+```
+
+We can try to check openfiles current limit:
```
Comment (zacharysarah) [inline README.md:78]: Avoid ["we"](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we):
```suggestion
Try checking the current limit for open files:
```

File: README.md [hunk 2]
```diff
@@ -58,6 +58,51 @@ make serve
...
+
+`launchctl limit maxfiles`
+
+And then we can do this:
```
Comment (zacharysarah) [inline README.md:82]:
```suggestion
Then run the following commands:
```

File: README.md [hunk 3]
```diff
@@ -58,6 +58,51 @@ make serve
...
+sudo mv limit.maxfiles.plist /Library/LaunchDaemons
+sudo mv limit.maxproc.plist /Library/LaunchDaemons
+
+sudo chown root:wheel /Library/LaunchDaemons/limit.maxfiles.plist
+sudo chown root:wheel /Library/LaunchDaemons/limit.maxproc.plist
+
+sudo launchctl load -w /Library/LaunchDaemons/limit.maxfiles.plist
+```
+
+This is working for Catalina as well as Mojave MacOS.
```
Comment (zacharysarah) [inline README.md:103]:
```suggestion
This works for Catalina as well as Mojave MacOS.
```

**Comment by irvifa** [comment]
@zacharysarah solved 👌

**Pull request merged**